contracts/vaults/RibbonThetaVault.sol

```diff
@@ -12,16 +12,27 @@ import {GnosisAuction} from "../protocols/GnosisAuction.sol";
 import {OptionsVaultStorage} from "../storage/OptionsVaultStorage.sol";
 import {IOtoken} from "../interfaces/GammaInterface.sol";
 import {IWETH} from "../interfaces/IWETH.sol";
-import {IStrikeSelection} from "../interfaces/IRibbon.sol";
-
-contract RibbonThetaVault is DSMath, GnosisAuction, OptionsVaultStorage {
+import {IGnosisAuction} from "../interfaces/IGnosisAuction.sol";
+import {
+    IStrikeSelection,
+    IOptionsPremiumPricer
+} from "../interfaces/IRibbon.sol";
+
+contract RibbonThetaVault is DSMath, OptionsVaultStorage {
+    using GammaProtocol for address;
```

Comment:
We can remove this, see: https://github.com/ribbon-finance/ribbon-v2/pull/2/files#r646754461